### DISCUSSION TITLE
Code Leak

### DISCUSSION THREAD

**Discussion by JulyFinal** [community/community#48365] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, our confidential code was accidentally leaked on Github, can we request that someone from the community contact each other to remove the code or have it destroyed.

**Comment by Akash1134** [comment, marked as answer]
Thank you kindly for reaching out. If you are experiencing behavior against our Terms of Service or need to report GitHub account-related abuse, you can reach out to our private support team. We are unable to assist with or discuss personal account details here in Discussions.

Please visit this link for more details: https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam We hope that helps!
